5.machine_learning/0.train_logistic_regression/README.md

```diff
@@ -0,0 +1,19 @@
+# Train logistic regression binary classifers
+
+In this submodule, we split the `Plate 4` data into training, testing, and holdout datasets and train binary logistic regression models to classify single-cells as either coming from healthy/nonfailing or failing hearts.
```

Comment:
Is there a way to link the plate 4 platemap here?

Reply:
Absolutely! Will do!